src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -206,6 +206,7 @@ void DlgPrefLibrary::slotResetToDefaults() {
     radioButton_dbclick_bottom->setChecked(false);
     radioButton_dbclick_top->setChecked(false);
     radioButton_dbclick_deck->setChecked(true);
+    checkBox_show_findonwebmenu->setChecked(true);
```

Comment:
Can you also remove this? I think the top menu entry does either not hurt, or can be hidden without a preference option if all FindOn menus are disabled.

Reply:
Oh okay, so no preference for the Find On Web

Reply:
Oh with the last changes we can not disable the FindOn menus.

So we can either not show FindOnWeb menu or show it with all services if we add this preference option.

If all the findon services are disabled (not possible for now), when the user hovers it on FindOnWeb menu, the menu turns to gray and freezes for a second.